src/main.js

```diff
@@ -119,7 +119,7 @@ let load = loader({
           instancePubKey: cfg.app.instancePubKey,
           state: state,
           regions: cfg.app.regions,
-          apiBaseUrl: cfg.server.baseUrl,
+          apiBaseUrl: cfg.server.publicUrl + '/v1',
```

Comment:
ahh, you're adding it here.  Let's leave this as a hard coded value in the config.

Reply:
This is bringing the approach into line with [taskcluster-queue](https://github.com/taskcluster/taskcluster-queue/blob/master/src/main.js#L408) and other services.  That service doesn't send the API base URL as context, but does include it as an argument to `v1.setup(..)`.